vllm/v1/core/kv_cache_manager.py

```diff
@@ -123,9 +123,16 @@ def get_computed_blocks(self,
 
         Returns:
             A tuple containing:
-                - A list of blocks that are computed for the request.
+                - A list of new blocks that are computed for the request.
                 - The number of computed tokens.
         """
+        # Request has already has its blocks, do not look up in block table.
```

Comment:
note: @WoosukKwon - this is the one part of the PR that I do not like. It breaks the previous invariant that the waiting loop has `request_id not in self.req_to_blocks`